lib/vm/src/trap/traphandlers.rs

```diff
@@ -29,6 +29,18 @@ use wasmer_types::TrapCode;
 // On Arm64, the udf alows for a 16bits values, so we'll use the same 0xC? to store the trapinfo
 static MAGIC: u8 = 0xc0;
 
+static mut DEFAULT_STACK_SIZE: usize = 1024 * 1024;
```

Comment:
Do we need to use a `static mut` variable here? If we make it an `AtomicUsize` we can drop the `unsafe`.

Reply:
Yeah, it's much cleaner to use Atomic, and I don't think it will have much impact on performances...